### PR TITLE
[NO-TICKET] Remove unused function attributes

### PR DESCRIPTION
**Pull request by cupakob**
#### Summary
Remove unused function attributes

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by agnivade — APPROVED** [review]
/cc @mahmud2011

File: app/file.go [hunk 1]
```diff
@@ -1036,7 +1036,7 @@ func (a *App) HandleImages(previewPathList []string, thumbnailPathList []string,
 
 			go func(img image.Image, path string, width int) {
```
Comment (streamer45) [inline app/file.go:1037]: Let's also remove `width` from here.

**Review by streamer45 — COMMENTED** [review]
Thank you @cupakob
I've left a suggestion for one additional change but otherwise looks good :+1:

File: app/file.go (hunk 1 above)
Comment (cupakob) [reply app/file.go]: fixed

**Review by streamer45 — APPROVED** [review]

**Pull request merged**